app/controllers/api/category_contents_controller.rb

```diff
@@ -0,0 +1,13 @@
+module API
+  class CategoryContentsController < APIController
+    def index
+      @primary_navigation, @secondary_navigation = Comfy::Cms::Category.navigation_categories
+      render json: @primary_navigation, scope: params[:locale].to_sym
```

Comment:
Cool, added some validation around the user input, also removed the to_sym since it wasn't really needed. Will also upgrade to 2.2.